lib/modules/newCommentCount.js

```diff
@@ -40,6 +40,19 @@ module.options = {
 		value: true,
 		description: 'Show the Subscribe button?',
 	},
+	monitorPostsVisited: {
+		type: 'boolean',
+		value: true,
+		description: 'Monitor the number of comments on posts you have visited.',
+		advanced: true,
+	},
+	monitorPostsVisitedIncognito: {
+		dependsOn: 'monitorPostsVisited',
+		type: 'boolean',
+		value: false,
```

Comment:
The thing is: most users don't care, they just expect it to work when they enable it. But like I said, I don't really care either, and these aren't really landmark features